dockerfile/rocm5.0-pytorch1.9.0.dockerfile

```diff
@@ -0,0 +1,134 @@
+    cd ucx-1.12.0 && \
+    mkdir build && cd build/ && \
+    ../contrib/configure-release --prefix=/opt/ucx && \
+    make -j $(nproc) && make install && \
+    rm -rf /tmp/ucx-1.12.0
+
+# Install OpenMPI
+ENV OPENMPI_VERSION=4.0.5
+RUN cd /tmp && \
+    wget -q https://www.open-mpi.org/software/ompi/v4.0/downloads/openmpi-${OPENMPI_VERSION}.tar.gz && \
+    tar xzf openmpi-${OPENMPI_VERSION}.tar.gz && \
+    cd openmpi-${OPENMPI_VERSION} && \
+    ./configure --enable-orterun-prefix-by-default && \
+    make -j $(nproc) all && \
+    make install && \
+    ldconfig && \
+    rm -rf /tmp/openmpi-${OPENMPI_VERSION}*
+
+# Install HPC-X
+RUN cd /opt && \
+    wget -q https://azhpcstor.blob.core.windows.net/azhpc-images-store/hpcx-v2.8.3-gcc-MLNX_OFED_LINUX-${OFED_VERSION}-ubuntu18.04-x86_64.tbz && \
+    tar xf hpcx-v2.8.3-gcc-MLNX_OFED_LINUX-${OFED_VERSION}-ubuntu18.04-x86_64.tbz && \
+    ln -s hpcx-v2.8.3-gcc-MLNX_OFED_LINUX-${OFED_VERSION}-ubuntu18.04-x86_64 hpcx && \
+    rm hpcx-v2.8.3-gcc-MLNX_OFED_LINUX-${OFED_VERSION}-ubuntu18.04-x86_64.tbz
+
+# Install Intel MLC
+RUN cd /tmp && \
+    mkdir -p mlc && \
+    cd mlc && \
+    wget --user-agent="Mozilla/5.0 (X11; Fedora; Linux x86_64; rv:52.0) Gecko/20100101 Firefox/52.0" https://www.intel.com/content/dam/develop/external/us/en/documents/mlc_v3.9a.tgz && \
+    tar xvf mlc_v3.9a.tgz && \
+    cp ./Linux/mlc /usr/local/bin/ && \
+    cd /tmp && \
+    rm -rf mlc
+
+# Install rccl-rdma-sharp-plugins
+ENV SHARP_VERSION=5.0
+RUN cd /opt/rocm && \
+    git clone -b release/rocm-rel-${SHARP_VERSION} https://github.com/ROCmSoftwarePlatform/rccl-rdma-sharp-plugins.git && \
+    cd rccl-rdma-sharp-plugins && \
+    ./autogen.sh && ./configure && make -j64
+
+# Install llvm+clang, required by hipify
+ENV LLVM_VERSION=13.0.0
+RUN cd /tmp && git clone -b llvmorg-${LLVM_VERSION} https://github.com/llvm/llvm-project.git && \
+    cd llvm-project/&& \
+    mkdir build && cd build && \
+    cmake -DCMAKE_INSTALL_PREFIX=/usr/lib/llvm-${LLVM_VERSION} -DLLVM_TARGETS_TO_BUILD="X86;NVPTX" -DLLVM_ENABLE_PROJECTS="clang"  -DCMAKE_BUILD_TYPE=Release ../llvm && \
+    make -j $(nproc) install && \
+    cd /tmp && rm -rf llvm-project
+
+# Install hipify
+ENV ROCM_VER=rocm-5.0.0
+RUN cd /tmp && git clone -b ${ROCM_VER} https://github.com/ROCm-Developer-Tools/HIPIFY.git && \
+    cd HIPIFY && \
+    mkdir build && cd build && \
+    cmake -DCMAKE_BUILD_TYPE=Release  -DCMAKE_INSTALL_PREFIX=/opt/rocm/hipify -DCMAKE_PREFIX_PATH=/usr/lib/llvm-${LLVM_VERSION} ..  && \
+    make -j $(nproc) install && \
+    cd /tmp && rm -rf HIPIFY
+
+ENV PATH="${PATH}:/opt/rocm/hip/bin/:/opt/rocm/hipify/" \
+    LD_LIBRARY_PATH="/opt/rocm/rccl-rdma-sharp-plugins/src/.libs:/usr/local/lib/:${LD_LIBRARY_PATH}" \
+    SB_HOME="/opt/superbench" \
+    SB_MICRO_PATH="/opt/superbench"
+
+WORKDIR ${SB_HOME}
+
+ADD third_party third_party
+RUN echo gfx90a >> /opt/rocm/bin/target.lst && ROCM_VERSION=${ROCM_VER} make -j -C third_party rocm
```

Comment:
Do we need to add `gfx90a` in the docker? If `ROCM_VERSION=${ROCM_VER}` work, why do we not implement it in other ROCm docker?